packages/vuetify/src/components/VDataTable/VRow.ts

```diff
@@ -0,0 +1,57 @@
+// Types
+import Vue, { VNode } from 'vue'
+import { PropValidator } from 'vue/types/options'
+import { TableHeader } from './mixins/header'
+
+// Utils
+import { getObjectValueByPath } from '../../util/helpers'
+
+export default Vue.extend({
+  name: 'v-row',
+
+  functional: true,
+
+  props: {
+    headers: Array as PropValidator<TableHeader[]>,
+    item: Object,
+    mobile: Boolean,
+    widths: Array as PropValidator<any[]>
+  },
+
+  render (h, { props, slots, data }): VNode {
+    const computedSlots = slots()
+
+    const columns = props.headers.map(header => {
+      const classes = {
+        [`text-xs-${header.align || 'left'}`]: true
+      }
+
+      const children = []
+      const value = getObjectValueByPath(props.item, header.value)
+
+      const scopedSlot = data.scopedSlots && data.scopedSlots[header.value]
+      const regularSlot = computedSlots[header.value]
+
+      if (scopedSlot) {
+        children.push(scopedSlot({ item: props.item, header, value, mobile: props.mobile }))
+      } else if (regularSlot) {
+        children.push(regularSlot)
+      } else {
+        if (props.mobile) {
+          children.push(h('div', { class: 'd-flex justify-content-between' }, [
```

Comment:
These are helper classes and will make it hard to overwrite in userland

Reply:
Yup, I know. A lot of this is still work in progress and/or placeholdery